multisig/proposals/arbitrumone/vip-013/index.ts

```diff
@@ -0,0 +1,223 @@
+import { parseUnits } from "ethers/lib/utils";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { arbitrumone } = NETWORK_ADDRESSES;
+
+export const COMPTROLLER_LIQUID_STAKED_ETH = "0x52bAB1aF7Ff770551BD05b9FC2329a0Bf5E23F16";
+
+export const wstETH = "0x5979D7b546E38E414F7E9822514be443A4800529";
+export const weETH = "0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe";
+export const WETH = "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1";
+
+export const VwstETH = "0x9df6B5132135f14719696bBAe3C54BAb272fDb16";
+export const VweETH = "0x246a35E79a3a0618535A469aDaF5091cAA9f7E88";
+export const VWETH = "0x39D6d13Ea59548637104E40e729E4aABE27FE106";
+
+export const PSR = "0xF9263eaF7eB50815194f26aCcAB6765820B13D41";
+export const vTokenReceiver = "0x5A9d695c518e95CD6Ea101f2f25fC2AE18486A61";
```

Comment:
This is the vToken receiver for the vwstETH tokens.

The token receiver for the vweETH tokens is `0x46cba1e9b1e5db32da28428f2fb85587bcb785e7`

Reply:
Refactored, b65668641d3c3d8bb1d533b80473ebccd781a8e3